api/src/main/java/ca/bc/gov/educ/eas/api/model/v1/SessionEntity.java

```diff
@@ -25,8 +25,8 @@ public class SessionEntity {
   @Column(name = "SESSION_ID", unique = true, updatable = false, columnDefinition = "BINARY(16)")
   private UUID sessionID;
 
-  @Column(name = "COURSE_SESSION", nullable = false, length = 6)
-  private String courseSession;
+  @Column(name = "SCHOOL_YEAR", nullable = false, length = 10)
+  private String schoolYear;
```

Comment:
we're thinking the format will be "2024/2025"? - if so, don't we want the length on this field to be 9?